src/savedialog.c

```diff
@@ -175,6 +175,28 @@ gint game_compare_func(GtkTreeModel *model, GtkTreeIter *a, GtkTreeIter *b, gpoi
 }
 
 
+static void save_load_dlg_response (GtkDialog * dlg, int response, gpointer user_data)
+{
+   switch (response)
+   {
+      case 100: /* save game */
```

Comment:
Does GTK define names for these constants?
They look undecipherable when used as anonymous magic constants.

Reply:
No, it's something you specify in `gtk_dialog_add_button(dialog, label, response)`

The "response" id is used in dialog response callback.

response cant be any GTK_RESPONSE_xxx but there is no GTK_RESPONSE_SAVE, so I specify an arbitrary number.